### DISCUSSION TITLE
Rendering blade conditionals should not require white space

### DISCUSSION THREAD

**Discussion by WordsofDefiance** [laravel/framework#45316] [Ideas]
When rendering conditionals in blade, this is valid:

```
@if ($condition1) TEXT A @elseif($condition2) TEXT B @else TEXT C @endif 
````

However, if any of the conditionals are not preceded by whitespace, that conditional statement will fail.

```
@if ($condition1) TEXT A@elseif($condition2) TEXT B @else TEXT C @endif 
```

Assuming `$condition1` is true, the above statement renders as `TEXT A@elseif($condition2) TEXT B`. 

I don't see a reason for this to act this way. Could it possibly be fixed so that conditional directives don't need to be surrounded by whitespace?

**Comment by parallels999** [comment]
Great idea, also what about writing without `;` on php sintax, we could write all on one line without white space
```php
$run=true echo 'Hola Mundo'if($run)echo 'First Run'
```
